EndlessClient/Rendering/Character/CharacterAnimator.cs

```diff
@@ -432,6 +431,11 @@ private CharacterRenderProperties FaceTarget(MapCoordinate characterCoord, MapCo
             return rp;
         }
 
+        public void ClearWalkPath()
+        {
+            _walkPath.Clear();
```

Comment:
Instead of exposing details of `_walkPath` outside of `CharacterAnimator`, how about clearing the walk path in the `Attack` call of `CharacterAnimator`? Is there a reason that wouldn't work that I'm not thinking of?

Reply:
I think you're right I'll have a look into this 😊